queue/queue.py

```diff
@@ -10,30 +10,37 @@ def __init__(self):
         """
         Initialize the Queue with an empty LinkedList
         """
-        pass
+        super().__init__()
+        self.items = LinkedList()
 
     def is_empty(self):
         """
         :returns: True if the LinkedList is empty and False if it isn't
         """
-        pass
+        if not self.items.head:
```

Comment:
This can be simplified to
```suggestion
        return self.items.head is not None:
```

Reply:
I want to understand this
so returning `self.items.head is not None` means true?
and  is `return False` necessary?

Reply:
The whole if statement is not needed